content/tutorial/tutorial.md

```diff
@@ -12,130 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Aquest tutorial no assumeix cap coneixement previ sobre React.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Abans de començar el tutorial {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+Anem a construir un petit joc durant aquest tutorial. **Hauràs estar temptat a obviar-perquè tu no estàs construint jocs en el dia a dia, però dóna-li una oportunitat.** Les tècniques que aprendràs en el tutorial són fonamentals per a construir qualsevol aplicació de React, i dominar-et donarà una entesa profunda de React.
 
 >Tip
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+>Aquest tutorial està dissenyat per a persones que prefereixen **aprendre fent**. Si tu prefereixes aprendre els conceptes des del principi, revisa la nostra [guia pas a pas](/docs/hello-world.html). Podries trobar aquest tutorial i la guia, complementàries entre si.
 
-The tutorial is divided into several sections:
+Aquest tutorial està dividit en diverses seccions:
 
-* [Setup for the Tutorial](#setup-for-the-tutorial) will give you **a starting point** to follow the tutorial.
-* [Overview](#overview) will teach you **the fundamentals** of React: components, props, and state.
-* [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
-* [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
+* [Configuració per a l'tutorial](#setup-for-the-tutorial) et donarà un punt de partida per seguir el tutorial.
+* [Visió general](#overview) t'ensenyarà **els fonaments** de React: components, props i estat.
+* [Completant el joc](#completing-the-game) t'ensenyarà **les tècniques més comuns** en desenvolupament de React.
+* [Afegint viatge en el temps](#adding-time-travel) et donarà una **visió més profunda** de les fortaleses úniques de React.
 
-You don't have to complete all of the sections at once to get the value out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
+No has de completar totes les seccions alhora per obtenir el valor d'aquest tutorial. Prova arribar tan lluny com puguis, fins i tot si és una o dues seccions.
```

Comment:
I think changing "No has de completar totes les seccions alhora per obtenir el valor d'aquest tutorial" to "No has de completar totes les seccions alhora per a poder treure partit del tutorial". In my opinion, "Prova arribar tan lluny com puguis, fins i tot si és una o dues seccions" it would be more accurate like "Prova d'arribar tan lluny com puguis, fins i tot si nomes són una o dues seccions".